curator/templates/page.html

```diff
@@ -1,27 +1,18 @@
 
 <!-- wp:template-part {"slug":"header","tagName":"header"} /-->
 
-<!-- wp:group {"layout":{"inherit":true}} -->
-<div class="wp-block-group">
-<!-- wp:post-title /-->
-</div>
-<!-- /wp:group -->
-
-<!-- wp:group {"tagName":"main"} -->
-<main class="wp-block-group">
-<!-- wp:post-featured-image {"align":"full"} /-->
-
-<!-- wp:post-content {"layout":{"inherit":true}} /-->
-</main>
-<!-- /wp:group -->
+<!-- wp:cover {"overlayColor":"foreground","minHeight":525,"minHeightUnit":"px","style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"margin":{"top":"0px","right":"0px","bottom":"0px","left":"0px"}}}} -->
+<div class="wp-block-cover" style="padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px;margin-top:0px;margin-right:0px;margin-bottom:0px;margin-left:0px;min-height:525px">
+<span aria-hidden="true" class="wp-block-cover__background has-foreground-background-color has-background-dim-100 has-background-dim"></span>
+<div class="wp-block-cover__inner-container">
+<!-- wp:cover {"useFeaturedImage":true,"dimRatio":40,"overlayColor":"foreground","minHeight":525,"minHeightUnit":"px","style":{"spacing":{"padding":{"top":"120px","right":"20px","bottom":"120px","left":"20px"}}}} -->
+<div class="wp-block-cover" style="padding-top:120px;padding-right:20px;padding-bottom:120px;padding-left:20px;min-height:525px"><span aria-hidden="true" class="wp-block-cover__background has-foreground-background-color has-background-dim-40 has-background-dim"></span><div class="wp-block-cover__inner-container"><!-- wp:post-title {"textAlign":"center","level":1,"style":{"spacing":{"margin":{"top":"0px","right":"0px","bottom":"0px","left":"0px"}}},"textColor":"background"} /--></div></div>
```

Comment:
I ended up using the [clamp value](https://github.com/Automattic/themes/blob/trunk/curator/theme.json#L63) for the left/right padding here in the single template, what do you think about using it here too?

Reply:
updated the values to clamp.